lib/core/src/lib/app-config/app-config.service.spec.ts

```diff
@@ -164,24 +164,23 @@ describe('AppConfigService', () => {
         expect(appConfigService.get('testUrl')).toBe('ftp://localhost:9090');
     });
 
-    it('should load external settings', () => {
-        appConfigService.load().then((config) => {
+    it('should load external settings', async () => {
+        await appConfigService.load().then((config) => {
```

Comment:
do we need to use `.then()` when we are using `await`?

```suggestion
        await appConfigService.load();
        expect(config).toEqual(mockResponse);
```